linkml/generators/fhirgen.py

```diff
@@ -0,0 +1,380 @@
+"""fhirgen: FHIR-related generators
```

Comment:
### Files modified when doing `poetry install`?
@hrshdhgd Hey Harshad. If you don't mind, I'd like to lean on you for support a little bit. I have a question. I did `poetry install`, and I noticed that lots of tests ran, and they changed many output test files. Is this supposed to happen? I think all the tests passed. Should I commit these files?

Reply:
@hrshdhgd never mind, i think this was the result of me running a docgen test